Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -35,6 +37,14 @@ public class ConsoleController : MonoBehaviour
 
     [SerializeField] private GameObject _toggleGodMode;
```

Comment:
From just looking at this variable I have on idea what game object it's referring to.

Reply:
^ Give it a more descriptive name for what it is